governator-legacy/build.gradle

```diff
@@ -8,4 +8,6 @@ dependencies {
     compile    'com.fasterxml.jackson.core:jackson-databind:2.4.3'
 
     testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'junit:junit:latest.release'
```

Comment:
It does. I'll make these a specific version so its consistent, but given that they are testCompile and not transitive I don't believe it makes much of a difference.